crates/swift-bridge-ir/src/bridged_type/shared_struct/struct_field/normalized_field.rs

```diff
@@ -38,6 +38,20 @@ impl NormalizedStructField {
         }
     }
 
+    /// Used when we want to avoid putting spaces at all between the field name and the colon.
```

Comment:
I think it would be clearer to do all of the field setting here instead of doing half of it here an half of it at the call site. 

Maybe something like:

```rust
/// Create a string for setting the value of a struct's field.
///
/// Example if named field -> "field_name: someValue".
/// Example if unnamed field -> "someValue".
struct_field_setter_string(&self, value: String) -> String {
    match &self.accessor {
        NormalizedStructFieldAccessor::Named(name) => {
            format!("{}: {}", name.to_string(), value)
        }
        NormalizedStructFieldAccessor::Unnamed(_idx) => {
            value
        }
    }
}

```